### DISCUSSION TITLE
Install

### DISCUSSION THREAD

**Discussion by VincentSong524** [coder/code-server#5109] [Q&A] [answered]
## How to install in Ubuntu ？
I can't install

**Comment by jsjoeio** [comment]
Have you tried following [the docs](https://coder.com/docs/code-server/latest/install#debian-ubuntu)?

**Comment by lg-nineoneninethreenine** [comment, marked as answer]
Simply run this:

```
curl -fsSL https://code-server.dev/install.sh | sudo bash
```


If it fails, it's sus for me since it has to work on all x86_64 machines. Maybe not Apple Silicon directly, I am highly unsure of that. But I could be wrong, hence if it fails, tell me the machine you run it on(system manufacturer and model) and some classy Neofetch details. To obtain that, run this:
```
sudo apt update # update packages to get latest apps
sudo apt install neofetch
neofetch
```
Share a screenshot of the neofetch output and I'll conjure something up if it is in the fail limit. Alert again: this is only _if you fail._ Else, you can start code-server directly.

**Reply by VincentSong524** [reply]
sws@10-60-90-193
----------------
OS: Ubuntu 20.04 LTS x86_64
Host: KVM UCLOUD 1.0.0 O PC (i440FX + PIIX, 1996)
Kernel: 5.4.0-48-generic
Uptime: 19 days, 8 hours, 15 mins
Packages: 677 (dpkg), 5 (snap) 
Shell: bash 5.0.16 
Resolution: 1024x768 
Terminal: /dev/pts/0 
CPU: AMD EPYC (1) @ 2.595GHz 
GPU: 00:02.0 Vendor 1234 Device 1111 
Memory: 211MiB / 1795MiB 


When i run this:

`curl -fsSL https://code-server.dev/install.sh | sudo bash` 

return: 

`curl: (28) Operation timed out after 300363 milliseconds with 0 out of 0 bytes received`

**Reply by lg-nineoneninethreenine** [reply]
You can happily download the DEB file from the releases page and download via that. That would be the same result.

**Reply by VincentSong524** [reply]
thanks for your reminder.

**Reply by lg-nineoneninethreenine** [reply]
Did it work?

**Reply by VincentSong524** [reply]
Now it's ready. Thank you.

**Comment by VincentSong524** [comment]
Yeah, thanks！

发自我的iPad
